modules/creating-custom-links.adoc

```diff
@@ -24,39 +24,6 @@ spec:
 ----
 <1> Valid location settings are `HelpMenu`, `UserMenu`, and `ApplicationMenu`.
```

Comment:
We'll want to add `NamespaceDashboard` to this list for 4.3

Reply:
@spadgett Thanks! I opened https://github.com/openshift/openshift-docs/pull/17714 to add it in.